src/wwwroot/controllers/dKimConfigCtrl.js

```diff
@@ -39,6 +39,9 @@
         vm.spfStatus = response.data.spf_ready; 
         vm.dKimPublicKey = 'k=rsa; p=' + response.data.dkim_public_key;
         vm.dKimSelector = response.data.dkim_selector + '._domainkey.' + vm.domain;
+        vm.trackingDomainStatus = response.data.tracking_domain_ready;
+        vm.trackingDomain = response.data.tracking_domain;
+        vm.trackingDomainCname = response.data.tracking_domain_cname;
```

Comment:
What do you think about renaming it as _canonical_tracking_domain_? or _expected_tracking_domain_cname_?

I mean, it is something that we require, it is not something that the client could decide or change, we are asking him to configure his domain in this way.